src/host/ut_host/CodepointWidthDetectorTests.cpp

```diff
@@ -19,7 +19,7 @@ static const std::vector<std::tuple<unsigned int, std::wstring, CodepointWidth>>
     { 0x7, L"\a", CodepointWidth::Narrow }, // BEL
     { 0x20, L" ", CodepointWidth::Narrow },
     { 0x39, L"9", CodepointWidth::Narrow },
-    { 0x414, L"\x414", CodepointWidth::Ambiguous }, // U+0414 cyrillic capital de
+    { 0x414, L"\x414", CodepointWidth::Narrow }, // U+0414 cyrillic capital de

```

Comment:
`widthDetector.GetWidth` won't ever return anything but `Narrow` or `Wide` from now on.

Reply:
Then what we have is no longer a unit test! That's a functional test ;(

Reply:
Eh, I'd say it's more like a conversion from a white-box to a black-box test. In this case it's fine though, since another test right after checks whether ambiguous characters result in the callback being called and being resolved correctly.